File: backend/src/performance/001-fibonacci/fib.rb

def fib(n)
    a, b = 0, 1
    (0..n).step(1) do |i|
        a, b = a + b, a
    end
    return a
end

if __FILE__ == $0
    arg = Integer(ARGV[0])
    fib(arg)
end

File: backend/src/performance/001-fibonacci/fib.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

from sys import argv

def fib(n):
    a, b = 0, 1
    for i in range(n+1):
        a, b = a + b, a
    return a

if __name__ == "__main__":
    arg = int(argv[1])
    fib(arg)